components/translation/custom_formats.rst

```diff
@@ -46,10 +46,10 @@ create the catalog that will be returned::
                 }
             }
 
-            $catalogue = new MessageCatalogue($locale);
-            $catalogue->add($messages, $domain);
+            $messageCatalogue = new MessageCatalogue($locale);
```

Comment:
Just wondering: if later we replace $catalogue by $translations ... maybe here we can replace $messageCatalogue by $translation? Maybe is not a good idea.

Reply:
Not sure really, the other place was importing the translations I thought that it was appropriate to rename it like that and here we are working with `$messageCatalogue`, I can change it but I am not sure?